hub/modules/config/configurations.bal

```diff
@@ -94,3 +94,12 @@ public configurable int KAFKA_CONSUMER_FETCH_MAX_BYTES = 3145728;
 
 # The maximum retry count
 public configurable int KAFKA_CONSUMER_MAX_PARTITION_FETCH_BYTES = 524288;
+
+# Kafka topic which is stores consolidated websub subscribers for this server
+public configurable string META_TOPICS = "registered-websub-topics,consolidated-websub-topics,registered-websub-subscribers,consolidated-websub-subscribers";
+
+# consolidator base url
+public configurable string CONSOLIDATOR_BASE_URL = "http://localhost:9192";
```

Comment:
Can we keep the consolidator pod name here ?

Reply:
done

Reply:
resolved